docs/modules/ROOT/pages/documentation/payara-server/h2/h2.adoc

```diff
@@ -30,4 +30,16 @@ To stop the database run the command below:
 asadmin> stop-database
 ----
 
-The above command will stop H2 by default. 
\ No newline at end of file
+The above command will stop H2 by default. 
+
+[[to-connect]]
+=== To connect to H2 DB databases
+
+To connect to an H2 DB database and execute SQL scripts, you should:
+
+- Stop Payara Server if running
+- Navigate to `/path/to/payara/payara5/h2db/bin` to find the h2db binaries
+- Execute `java -jar h2.jar` (this will start the H2 Admin UI server and open browser with the Admin UI URL)
+- In the browser, set JDBC URL to `jdbc:h2:/path/to/payara/payara5/glassfish/domains/domain1/lib/databases/ejbtimer` to connect to the EJB timer database (or change the path to connect to another DB)
```

Comment:
reference to `domain1` which is not recommended for production. make it generic like `<domain-name>`?

Reply:
Thanks, done.